src/renderer/settings/modals/websearch-editing-modal-component.ts

```diff
@@ -121,7 +132,7 @@ export const websearchEditingModal = Vue.extend({
                             {{ translations.websearchUrl }}
                         </label>
                         <div class="control is-expanded">
-                            <input class="input" type="url" v-model="websearchEngine.url" :placeholder="getUrlPlaceholder()">
+                            <input class="input" type="url" v-model="websearchEngine.url" :placeholder="getUrlPlaceholder()" v-on:blur="onUrlBlurGetFaviconUrl()">
```

Comment:
Can we use `@blur` instead of `v-on:blur`?

Reply:
Updated to use `@blur` instead.